concent_api/core/message_handlers.py

```diff
@@ -1305,17 +1306,14 @@ def handle_send_subtask_results_verify(
     compute_task_def = task_to_compute.compute_task_def
     requestor_public_key = hex_to_bytes_convert(task_to_compute.requestor_public_key)
     provider_public_key = hex_to_bytes_convert(task_to_compute.provider_public_key)
-
     validate_golem_message_subtask_results_rejected(subtask_results_rejected)
-    validate_that_golem_messages_are_signed_with_key(
```

Comment:
Why is this removed ?

Reply:
O, if I am to be honest I don't know. It might be deleted during some of rebeases, but I'm not sure. Of course restored.